test/e2e/cluster_upgrade.go

```diff
@@ -48,6 +48,11 @@ var _ = framework.KubeDescribe("Upgrade [Feature:Upgrade]", func() {
 			cm.Register(func(sem *chaosmonkey.Semaphore) {
 				// Close over f.
 				testServiceRemainsUp(f, sem)
+				testSecretsDuringUpgrade(f, sem)
```

Comment:
You'll either need to change the test description in the above "It()" clause, or split these into separate "It()" clauses, each with their own description.  I strongly prefer the latter option, as it makes things far more transparent as to what's failing.

I realize that that will be difficult in practise, as we would need to figure out how best to avoid a separate cluster upgrade per test.  Happy to leave this change to a separate PR if that makes more sense.